packages/ui/src/components/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { ToggleGroup, ToogleGroupOption } from "./ToggleGroup";
+import { useState } from "react";
+import { fn } from "@storybook/test";
```

Comment:
Minor, non-blocking:
We're not using this dependency, IMHO it's safe to remove

Reply:
Yes. thanks.